### PR TITLE
refactor(core): Make executions pruning settings configurable

### PR DESCRIPTION
**Pull request by ivov**

### CODE REVIEW

File: packages/cli/src/config/schema.ts [hunk 1]
```diff
@@ -316,16 +316,30 @@ export const schema = {
 			env: 'EXECUTIONS_DATA_PRUNE',
 		},
 		pruneDataMaxAge: {
-			doc: 'How old (hours) the finished execution data has to be to get deleted',
+			doc: 'How old (hours) the finished execution data has to be to get soft-deleted',
 			format: Number,
 			default: 336,
 			env: 'EXECUTIONS_DATA_MAX_AGE',
 		},
-		pruneDataInterval: {
```
Comment (ivov) [inline packages/cli/src/config/schema.ts:324]: Added by #7439 which was merged prematurely while the conversation was ongoing. Unreleased so it should be fine to modify without marking as a breaking change.

Comment (krynble) [reply packages/cli/src/config/schema.ts:324]: It was actually released yesterday in version 1.12.0, according to the last comment on the mentioned PR

**Review by krynble — APPROVED** [review]
LGTM

File: packages/cli/src/config/schema.ts (hunk 1 above)
Comment (netroy) [reply packages/cli/src/config/schema.ts:324]: Sorry. I merged it because it was approved, and it sounded like it was blocking a customer.

Comment (ivov) [reply packages/cli/src/config/schema.ts:324]: No worries, I should've marked as a draft.

**Comment by janober** [comment]
Got released with [`n8n@1.14.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.14.0)

**Pull request merged**